include/openenclave/attestation/verifier.h

```diff
@@ -0,0 +1,192 @@
+
+/**
+ * oe_verifier_initialize
+ *
+ * Initializes the verifier environment configured for the platform and
+ * the calling application.
+ *
+ * This function is idempotent and can be called multiple times without
+ * adverse effect.
+ *
+ * @experimental
+ *
+ * @retval OE_OK on success.
+ * @retval other appropriate error code.
+ */
+oe_result_t oe_verifier_initialize(void);
+
+/**
+ * oe_verifier_get_formats
+ *
+ * Gets a list of evidence formats accepted by the verifier for evidence
+ * verification.
+ *
+ * @experimental
+ *
+ * @param[out] formats An output poionter that will be assigned the address of
+ * a dynamically allocated buffer that holds the returned list of formats
+ * supported for evidence verification
+ * @param[out] formats_length A pointer that points to the length of the
+ * returned formats list (number of format ID entries).
+ * @retval OE_OK on success.
+ * @retval OE_INVALID_PARAMTER At least one of the parameters is invalid.
+ * @retval other appropriate error code.
+ */
+oe_result_t oe_verifier_get_formats(
+    oe_uuid_t** formats,
+    size_t* formats_length);
+
+/**
+ * oe_verifier_free_formats
+ *
+ * Frees a formats list.
+ *
+ * @experimental
+ *
+ * @param[in] formats The formats list.
+ * @retval OE_OK The function succeeded.
+ * @retval other appropriate error code.
+ */
+oe_result_t oe_verifier_free_formats(oe_uuid_t* formats);
+
+/**
+ * oe_verifier_get_format_settings
+ *
+ * Gets the optional settings data for the input evidence format.
+ *
+ * @experimental
+ *
+ * @param[in] format The format for which to retrieve the optional settings.
+ * @param[out] settings An output pointer that will be assigned the address of
+ * a dynamically allocated buffer that holds the returned settings data. This
+ * pointer will be assigned a NULL value if there is no settings needed.
+ * @param[out] settings_size A pointer that points to the size of the returned
+ * format settings buffer (number of bytes).
+ * @retval OE_OK on success.
+ * @retval OE_INVALID_PARAMTER At least one of the parameters is invalid.
+ * @retval other appropriate error code.
+ */
+oe_result_t oe_verifier_get_format_settings(
+    const oe_uuid_t* format,
+    uint8_t** settings,
+    size_t* settings_size);
+
+/**
+ * oe_verifier_free_format_settings
+ *
+ * Frees a format settings buffer.
+ *
+ * @experimental
+ *
+ * @param[in] settings The format settings buffer.
+ * @retval OE_OK The function succeeded.
+ * @retval other appropriate error code.
+ */
+oe_result_t oe_verifier_free_format_settings(uint8_t* settings);
+
+/**
+ * oe_verify_evidence
+ *
+ * Verifies the attestation evidence and returns well known and custom claims.
+ * This is available in the enclave and host.
+ *
+ * The following base claims will be returned at the minimum:
+ *
+ *  - id_version (uint32_t)
+ *      - Version number. Must be 1.
+ *  - security_version (uint32_t)
+ *      - Security version of the enclave. (ISVN for SGX).
+ * - attributes (uint64_t)
```

Comment:
nit: make spacing match lines 114 and 116

Reply:
Thanks!